setup.py

```diff
@@ -5,5 +5,7 @@
     name='pylibui',
     version='0.0.1',
     description='Python wrapper for libui',
-    packages=find_packages()
+    packages=find_packages(),
+    package_data={'pylibui.libui.sharedlibs': '*'},
+    include_pacakge_data=True,
```

Comment:
Isn't "pacakge" mispelled? It may be this that is failing the CI..

Reply:
Yes, I misspelled it. (I make an amend commit) But I check the log. It's because the test need to `mkdir pylibui/libui/sharedlibs` where I have to put an `__init__.py`. I've add an `if` for it.